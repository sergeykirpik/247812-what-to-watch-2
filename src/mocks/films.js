const films = [
  {
    id: 1,
    title: `Fantastic Beasts: The Crimes of Grindelwald`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  },
  {
    id: 2,
    title: `Bohemian Rhapsody`,
    previewImage: `img/bohemian-rhapsody.jpg`,
  },
  {
    id: 3,
    title: `Macbeth`,
    previewImage: `img/macbeth.jpg`,
  },
  {
    id: 4,
    title: `Aviator`,
    previewImage: `img/aviator.jpg`,
  },
  {
    id: 5,
    title: `We need to talk about Kevin`,
    previewImage: `img/we-need-to-talk-about-kevin.jpg`,
  },
  {
    id: 6,
    title: `What We Do in the Shadows`,
    previewImage: `img/what-we-do-in-the-shadows.jpg`,
  },
  {
    id: 7,
    title: `Revenant`,
    previewImage: `img/revenant.jpg`,
  },
  {
    id: 8,
    title: `Johnny English`,
    previewImage: `img/johnny-english.jpg`,
  },
  {
    id: 9,
    title: `Shutter Island`,
    previewImage: `img/shutter-island.jpg`,
  },
  {
    id: 10,
    title: `Pulp Fiction`,
    previewImage: `img/pulp-fiction.jpg`,
  },
  {
    id: 11,
    title: `No Country for Old Men`,
    previewImage: `img/no-country-for-old-men.jpg`,
  },
  {
    id: 12,
    title: `Snatch`,
    previewImage: `img/snatch.jpg`,
  },
  {
    id: 13,
    title: `Moonrise Kingdom`,
    previewImage: `img/moonrise-kingdom.jpg`,
  },
  {
    id: 14,
    title: `Seven Years in Tibet`,
    previewImage: `img/seven-years-in-tibet.jpg`,
  },
  {
    id: 15,
    title: `Midnight Special`,
    previewImage: `img/midnight-special.jpg`,
  },
  {
    id: 16,
    title: `War of the Worlds`,
    previewImage: `img/war-of-the-worlds.jpg`,
  },
  {
    id: 17,
    title: `Dardjeeling Limited`,
    previewImage: `img/dardjeeling-limited.jpg`,
  },
  {
    id: 18,
    title: `Orlando`,
    previewImage: `img/orlando.jpg`,
  },
  {
    id: 19,
    title: `Mindhunter`,
    previewImage: `img/mindhunter.jpg`,
  },
  {
    id: 20,
    title: `Midnight Special`,
    previewImage: `img/midnight-special.jpg`,
  }
];

export default films;
