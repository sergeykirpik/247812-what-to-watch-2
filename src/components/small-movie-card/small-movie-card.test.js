import React from 'react';
import renderer from 'react-test-renderer';
import SmallMovieCard from './small-movie-card.jsx';

import films from '../../mocks/films';

it(`renders correctly`, () => {
  const tree = renderer.create(
      <SmallMovieCard
        film={films[0]} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
