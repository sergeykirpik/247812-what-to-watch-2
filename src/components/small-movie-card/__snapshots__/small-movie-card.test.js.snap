// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      height={175}
      muted={true}
      poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      width={280}
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/details?1"
      onClick={[Function]}
    >
      Fantastic Beasts: The Crimes of Grindelwald
    </a>
  </h3>
</article>
`;
